apm-sniffer/apm-sdk-plugin/undertow-plugins/undertow-1.x-plugin/readme.md

```diff
@@ -0,0 +1,21 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+### undertow 1.x plugin
+Copy the undertow-2.x-plugin plugin from skywalking 6.0.0
+
+Passed the test in skywalking 6.5.0-SNAPSHOT version and undertow 1.4.27 version
```

Comment:
This document is strange, why do we need this? If we accept a plugin for new version, you need to change the existing one. And this plugin must pass the tests for multiple versions.

Remove this file, please. And submit another pr to move undertow plugin test into this repo. 
Last step, make the existing old plugin passed 1.x versions. Notice, onr version is not enough.